### PR TITLE
Update 4.1.1 Parsing to address AG WG concerns

### PR DESCRIPTION
**Pull request by maryjom**
See the [TF Resolution on 4.1.1 Parsing](https://www.w3.org/2024/04/04-wcag2ict-minutes#r03).

### CODE REVIEW

**Review by ChrisLoiselle — APPROVED** [review]

**Review by ChrisLoiselle — APPROVED** [review]

File: comments-by-guideline-and-success-criterion.md [hunk 1]
```diff
@@ -1063,15 +1063,21 @@ WCAG 2.2 has made this success criterion obsolete and removed it as a requiremen
...
 
->This Success Criterion should be considered as always satisfied for any content using HTML or XML.
+>Success Criterion 4.1.1 was originally adopted to address problems that assistive technology had directly parsing HTML. Since this criterion was written, the HTML Standard has adopted specific requirements governing how user agents must handle incomplete tags, incorrect element nesting, duplicate attributes, and non-unique IDs.
+>
+>Although the HTML Standard treats some of these cases as non-conforming for authors, it is considered to "allow these features" for the purposes of this Success Criterion because the specification requires that user agents support handling these cases consistently. In practice, this criterion no longer provides any benefit to people with disabilities in itself.
+>
+>Issues such as missing roles due to inappropriately nested elements or incorrect states or names due to a duplicate ID are covered by different Success Criteria and should be reported under those criteria rather than as issues with 4.1.1.
+>
+>This criterion should be considered as always satisfied for any content using HTML or XML.
```
Comment (WilcoFiers) [inline comments-by-guideline-and-success-criterion.md:1072]: The crux of the problem is here. Yes, for HTML and XML this criterion is always satisfied. But not every markup language is HTML / XML based. Far from it. I think the 4.1.1 content from the previous WCAG2ICT is still relevant here.

**Review by WilcoFiers — CHANGES_REQUESTED** [review]

File: comments-by-guideline-and-success-criterion.md (hunk 1 above)
Comment (ChrisLoiselle) [reply comments-by-guideline-and-success-criterion.md:1072]: @maryjom did you want to add in @WilcoFiers' edits or would you like me to approve his request?

Comment (maryjom) [reply comments-by-guideline-and-success-criterion.md:1072]: @ChrisLoiselle This is an unresolved discussion. I am meeting with Wilco and Dan tomorrow morning on this to decide what to do. The 2013 WCAG2ICT content that is suggested to be added has a lot of outdated content in it. I still have concerns with requiring this SC.

**Review by ChrisLoiselle — APPROVED** [review]

**Review by ChrisLoiselle — APPROVED** [review]

**Review by ChrisLoiselle — APPROVED** [review]

**Review by ChrisLoiselle — DISMISSED** [review]

File: comments-by-guideline-and-success-criterion.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ Comments by Guideline and Success Criterion
 <div class="ednote">
 The WCAG2ICT Task Force has incorporated all of the new WCAG 2.1 and WCAG 2.2 guidelines, success criteria, and glossary terms into this draft. The Task Force also made changes to address public comments.</div>
 
-The sections that follow are organized according to the principles, guidelines, and success criteria from WCAG 2. The text of each success criterion from WCAG 2 is copied as quoted text. Following that, the WCAG2ICT guidance is provided. The WCAG2ICT guidance can be found in the sections where the headings begin with "Applying..." to highlight that this is the content specific to this document.
+The sections that follow are organized according to the principles, guidelines, and succxess criteria from WCAG 2. The text of each success criterion from WCAG 2 is copied as quoted text. Following that, the WCAG2ICT guidance is provided. The WCAG2ICT guidance can be found in the sections where the headings begin with "Applying..." to highlight that this is the content specific to this document.
```
Comment (maryjom) [inline comments-by-guideline-and-success-criterion.md:6]:
```suggestion
The sections that follow are organized according to the principles, guidelines, and success criteria from WCAG 2. The text of each success criterion from WCAG 2 is copied as quoted text. Following that, the WCAG2ICT guidance is provided. The WCAG2ICT guidance can be found in the sections where the headings begin with "Applying..." to highlight that this is the content specific to this document.
```

**Review by ChrisLoiselle — APPROVED** [review]

**Review by daniel-montalvo — APPROVED** [review]

**Pull request merged**